zproject/backends.py

```diff
@@ -108,6 +108,50 @@ def common_get_active_user(email: str, realm: Realm,
         return None
     return user_profile
 
+def generate_dev_ldap_dir(mode: str, extra_users: int=0) -> Dict[str, Dict[str, Sequence[str]]]:
```

Comment:
`zerver/lib/test_fixtures.py` might be a better home for this function.

Reply:
Just a thought, the function test_fixtures.py generates a fixture file and tests it against the existing fixture file. The existing fixture file is generally used in the app somewhere.
While in the case of `generate_dev_ldap_dir`, the fixture files are for testing whether the function returns the correct dict. Also the fixture files in our case are used only in the tests and not anywhere else. I agree that `zproject/backends.py` is not the best place to put that function, but I think `zerver/lib/test_fixtures.py` is also a not better home for the function.